dex/networks/btc/script_test.go

```diff
@@ -424,7 +424,7 @@ func TestInputInfo(t *testing.T) {
 	if err == nil {
 		t.Fatalf("no error for missing redeem script")
 	}
-	// Redeem script must be parseable.
+	// Redeem script must be parsable.
```

Comment:
Not sure about this one either, but I think "parseable" is way more common in programming context.

Reply:
Yeah, just a false positive typo. Didn't notice.